cpp/include/utilities/cython.hpp

```diff
@@ -22,65 +22,55 @@
 namespace cugraph {
 namespace cython {
 
-enum class numberTypeEnum : int { intType, floatType, doubleType };
+enum class numberTypeEnum : int { int32Type, int64Type, floatType, doubleType };
 
-// FIXME: The GraphC??View* types will not be used in the near future. Those are
-// left in place as cython wrappers transition from the GraphC* classes to
-// graph_* classes. Remove GraphC* classes once the transition is complete.
 enum class graphTypeEnum : int {
+  // represents unintiialized or NULL ptr
   null,
+  // represents some legacy Cxx type. This and other LegacyCxx values are not
+  // used for the unique_ptr in a graph_container_t, but instead for when this
+  // enum is used for determining high-level code paths to take to prevent
+  // needing to expose each legacy enum value to cython.
+  LegacyCSR,
+  LegacyCSC,
+  LegacyCOO,
+  // represents that a GraphCxxView* unique_ptr type is present in a
+  // graph_container_t.
   GraphCSRViewFloat,
   GraphCSRViewDouble,
   GraphCSCViewFloat,
   GraphCSCViewDouble,
   GraphCOOViewFloat,
   GraphCOOViewDouble,
```

Comment:
`LegacyCSR`-`LegacyCOO` represent legacy types that are CSR, CSC, etc., but don't represent anything stored in the container.  Those enum values are used for passing to `populate_graph_container` to tell it to create either a `GraphCSRView`, `GraphCSCView`, etc.  I just needed a flag value to tell the function which view type to create, and the other number type enum values passed in determine if it'll be a `GraphCSRViewDouble` vs. `GraphCSRViewFloat` etc.

The enum values `GraphCSRViewFloat`, etc. represent specific legacy types, and those enum values are used to determine what pointer type is stored in the container.

I had `LegacyCOO` values in a separate enum from `GraphCSRViewDouble` values prior to this refactoring, but figured I could have one enum type for all graph types either abstract or concrete, but maybe it's less clear now?  Also, this enum will go away once we remove the legacy classes.

Reply:
Got it thanks for the explanation, if GraphCSRViewFloat - GraphCOOViewDouble are also legacies, I guess naming those as LegacyGraphCSRViewFloat - LegacyGraphCOOViewFloat may make things clearer, but if this will be short-lived, then I may better not bother. I think this will become much clearer once the legacy part retires.